config/webpack/environment.js

```diff
@@ -0,0 +1,3 @@
+const { environment } = require('@rails/webpacker')
+
+module.exports = environment
```

Comment:
Typically, you'd set up `webpack` using a config file [like this](https://webpack.js.org/configuration/).

Webpacker apparently takes care of this for us - all of the relevant config options are now in `config/webpacker.yml`, and they get translated into JS in this file.